src/options/JsonConfig.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useRef, useState } from "react";
+import PropTypes from "prop-types";
+import parserBabylon from "prettier/parser-babylon";
+import prettier from "prettier/standalone";
+import { validateOptions } from "./options";
+
+const SAVED_TIMEOUT = 500;
+const STRINGIFY_SPACING = 2;
+
+export default function JsonConfig({ errors, options, setErrors, setOptions }) {
+  const textareaEl = useRef(null);
+  const [textAreaVal, setTextAreaVal] = useState(
+    JSON.stringify(options),
+    null,
+    STRINGIFY_SPACING
```

Comment:
Looks like the closing parent is in the wrong place: `null` and `STRINGIFY_SPACING` should be arguments for `JSON.stringify` rather than `useState`

Reply:
Ha, good catch